src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -176,7 +176,7 @@ inline bool IsShaderContraintsMet(const int R,
         if(!(0 <= params.GetBackwardPadH() && params.GetBackwardPadH() < std::pow(2, 16)))
             return false;
     }
-    const auto grid_workgroup_count_x = params.GetStream().GetMaxComputeUnits();
+    const auto grid_workgroup_count_x = params.GetStream().GetMaxHardwareComputeUnits();
```

Comment:
Is it so that HW CUs is required only for this Solver or we shall expect more changes in the future?

Reply:
In other solvers, either the gfx10 is not supported, or HW CU is not. So HW CUs not used anywhere else at the moment.